contracts/impls/trustable/WitnetRequestBoardTrustableOMGX.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+
+/* solhint-disable var-name-mixedcase */
+
+pragma solidity >=0.7.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+// Inherits from:
+import "./WitnetRequestBoardTrustableBase.sol";
+
+// Uses:
+import "../../interfaces/IERC20.sol";
+
+/// @title Witnet Request Board OVM-compatible (Optimism) "trustable" implementation.
+/// @notice Contract to bridge requests to Witnet Decentralized Oracle Network.
+/// @dev This contract enables posting requests that Witnet bridges will insert into the Witnet network.
+/// The result of the requests will be posted back to this contract by the bridge nodes too.
+/// @author The Witnet Foundation
+contract WitnetRequestBoardTrustableOMGX
+    is
+        Payable,
+        WitnetRequestBoardTrustableBase
+{
+    uint256 internal lastBalance;
+    uint256 internal immutable _OVM_GAS_PRICE;
+
+    modifier ovmPayable {
```

Comment:
Since its only used once, maybe the modifier is no needed. Anyhow, could you add some comment?

Reply:
In fact, the `upgradeQuery` should alse be overriden by imposing call to this modifier, so it's actually used twice in the same contract. Besides, it could be reused by an eventual new OMGX implementation inheriting from this one.

Reply:
Comment added :-)